requirements.txt

```diff
@@ -1,2 +1 @@
-mkdocs==1.2.3
-jinja2==3.0.3  # Downpin else mkdocs breaks
+mkdocs==1.6.0
```

Comment:
mkdocs default template has a lot of a11y issues in 1.6.0, let's hold off until 1.6.1 is out…

Reply:
Updated to v1.6.1 ✅